tests/runtime/lists/test.cpp

```diff
@@ -12,28 +12,28 @@ static bool equal(T const&a, S const& b) {
     return a == b;
 }
 template<class R, class S>
-static bool equal(wit::span<R> const&a, wit::span<S> const& b) {
+static bool equal(std::span<R> const&a, std::span<S> const& b) {
     if (a.size() != b.size()) { return false; }
     for (uint32_t i = 0; i<a.size(); ++i) {
         if (!equal(a[i], b[i])) { return false; }
     }
     return true;
 }
 template<class R>
-static bool equal(wit::vector<R> const&a, wit::span<R> const& b) {
+static bool equal(wit::vector<R> const&a, std::span<R> const& b) {
     return equal(a.get_view(), b);
 }
 template<class R>
-static bool equal(wit::span<const R> const&a, wit::vector<R> const& b) {
+static bool equal(std::span<const R> const&a, wit::vector<R> const& b) {
     return equal(b, a);
 }
 template<class R>
-static bool equal(wit::span<const R> const&a, std::vector<R> const& b) {
-    return equal(a, wit::span<R>(b));
+static bool equal(std::span<const R> const&a, std::vector<R> const& b) {
+    return equal(a, std::span<const R>(b));
 }
 template<class R>
 static bool equal(wit::vector<R> const&a, std::vector<R> const& b) {
-    return equal(a.get_view(), wit::span<R>(b));
+    return equal(a.get_view(), std::span<R>(b));
```

Comment:
more "std::", less language fragmentation (as long as you can use a new enough compiler ... I will ask people working on MISRA :wink:)